modules/running-cluster-oci-agent-based.adoc

```diff
@@ -0,0 +1,41 @@
+
+.Procedure
+
+. Upload the Agent ISO image to Oracle’s default Object Storage bucket and then import the Agent ISO image as a custom image to this bucket. You must then configure the custom image to boot in Unified Extensible Firmware Interface (UEFI) mode. See link:https://docs.oracle.com/en-us/iaas/secure-desktops/create-custom-image.htm[Creating a custom image] and link:https://docs.oracle.com/en-us/iaas/Content/Compute/Tasks/configuringimagecapabilities.htm#ariaid-title5[Using the Console] in Oracle’s documentation.
++
+For example, from **Compute > Custom images**, import the Agent ISO image to the bucket, and enter values in the following fields:
++
+* Name: *oci-cluster*
+* Bucket: Select the bucket where you uploaded the discovery ISO image
+* Object name: Select the name of the discovery ISO
+* Image type: *QCOW2*
++
+After the image imports, go to the **Edit image capabilities** setting and ensure  that `UEFI_64` is selected for the **Firmware** field. 
+
+. Create a compute instance from the supplied base image for your preferred cluster topology, such as a single-node OpenShift (SNO) cluster, a highly-availability cluster that contains a minimum of 3 control plane instances and two compute instances, or a compact three-node cluster that contains a minimum of three control plane instances. See link:https://docs.oracle.com/en-us/iaas/Content/Compute/Tasks/launchinginstance.htm#top[Creating an instance] (Oracle documentation).
++
+[IMPORTANT]
+====
+Before you create the compute instance, check that you have enough memory and disk resources for your cluster. See xref:../../installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.html#recommended-resources-for-topologies[Recommended resources for topologies].  Additionally, ensure that at least one compute instance has the same IP address as the address stated under `rendezvousIP` in the `agent-config.yaml` file.
+====
++
+The following example lists important settings for an instance named *oci-cluster-master*. 
++
+* Go to **Image and shape section >  Image >  My images** and then select your custom image.  
+* Go to  **Image and shape section > Shape menu** and then select at least 4 CPUs and 16 GB of memory.
+* In **Primary VNIC IP addresses > Primary network**, select a VCN, such as *oci-cluster-vcn*. From the **Subnet** section, select your subnet, such as *ici-cluster-private-subnet*. For public IPV4 subnets, ensure that you select the **Do not assign a public IPv4 address** checkbox.
+* Click **Show advanced options**. Select the **Use network security groups** to control traffic checkbox, and then set your NSG, such as *oci-cluster-controlplane-nsg*.
+* From the **Boot volume** section, select the **Specify a custom boot volume size** checkbox. Enter a value that is at least `100` GB for the boot volume size. Allocate the number of VPUs for your organization needs, such as `100`. 
```

Comment:
I suggest including example names in the preceding steps where we discuss the creation of the OCI network. This will provide users with context on where the names are derived from. Also, if we are using the example volume of 100 GB of the test environment, we should also use example VPUs, such as 20 or 30.

Reply:
Thanks, updated the VPU figure.